### PR TITLE
README add actual documentation

### PR DESCRIPTION
**Pull request by petertseng**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,7 @@
-# testrepo
\ No newline at end of file
+# testrepo
+
+LOREM IPSUM
```
Comment (petertseng) [inline README.md]: Comment on the first LOREM IPSUM

File: README.md [hunk 2]
```diff
@@ -1 +1,7 @@
...
+
+This is a good README.
+
+LOREM IPSUM
```
Comment (petertseng) [inline README.md]: comment on the SECOND lorem ipsum

File: README.md [hunk 3]
```diff
@@ -1 +1,13 @@
...
\ No newline at end of file
+# testrepo
+
+LOREM IPSUM
+
+This is a good README.
+
+This project does A THING.
+
+LOREM IPSUM
```
Comment (petertseng) [inline README.md]: comment on the SECOND lorem ipsum

File: README.md [hunk 4]
```diff
@@ -1 +1,21 @@
...
+
+Surrounding lines probably need to be the same (after)
+
+This is a good README.
+
+This project does A THING.
+
+Surrounding lines probably need to be the same (before)
+
+LOREM IPSUM
```
Comment (petertseng) [inline README.md]: I hereby comment on the second LOREM IPSUM

File: README.md [hunk 5]
```diff
@@ -1 +1,21 @@
-# testrepo
\ No newline at end of file
+# testrepo
+
+Surrounding lines probably need to be the same (before)
+
+LOREM IPSUM
```
Comment (mpuncel) [inline README.md]: this one

File: README.md [hunk 6]
```diff
@@ -1 +1,29 @@
...
+
+This is a good README.
+
+This project does A THING.
+
+Surrounding lines probably need to be the same (before)
+Surrounding lines probably need to be the same (before)
+Surrounding lines probably need to be the same (before)
+
+LOREM IPSUM
```
Comment (petertseng) [inline README.md]: I hereby comment on the **second** LOREM IPSUM

File: README.md (hunk 4 above)
Comment (mpuncel) [inline README.md]: that one

File: README.md [hunk 7]
```diff
@@ -1 +1,25 @@
...
+Surrounding lines probably need to be the same (after)
+
+This is a good README.
+
+This project does A THING.
+
+Surrounding lines probably need to be the same (before)
+Surrounding lines probably need to be the same (before)
+
+LOREM IPSUM
```
Comment (petertseng) [inline README.md]: I hereby comment on the **second** LOREM IPSUM

Comment (petertseng) [inline README.md]: I hereby comment on the **second** LOREM IPSUM

Comment (petertseng) [inline README.md]: second lorem ipsum, many months later.

Comment (petertseng) [inline README.md]: second lorem ipsum many months later - as part of a review.

Comment (petertseng) [inline README.md]: the comment now correctly appears on the second lorem ipsum!!! amazing.

**Pull request closed without merging**